kitchen.global.yml

```diff
@@ -3,9 +3,6 @@ provisioner:
   attributes:
     cluster:
       # right now tests depend on this parameter: we will try to remove the dependency later
-      base_os: alinux
+      base_os: alinux2
```

Comment:
Shouldn't we remove it?

Reply:
In another PR, I'm going to move most of the configuration from `kitchen.global.yml` into other kitchen file.

Reply:
Done